src/parser/metrics/CommentLines.ts

```diff
@@ -37,21 +39,12 @@ export class CommentLines implements Metric {
         const query = queryBuilder.build();
         const matches = query.matches(tree.rootNode);
 
-        const commentLines = matches.reduce((accumulator, match) => {
-            const captureNode = match.captures[0].node;
-
-            if (
-                this.isFileHeaderComment(captureNode) ||
-                this.isCommentFollowedByClass(captureNode)
-            ) {
-                return accumulator;
-            }
-
-            const commentLinesWithText = this.findTextLines(captureNode.text).length;
-            const docBlockLines = this.findDocBlockLines(captureNode.text).length;
-
-            return accumulator + commentLinesWithText - docBlockLines;
-        }, 0);
+        let commentLines = 0;
+        for (const match of matches) {
+            const captureNode: SyntaxNode = match.captures[0].node;
+            commentLines =
+                commentLines + 1 + (captureNode.endPosition.row - captureNode.startPosition.row);
```

Comment:
What about:

```
if (answer === 42 /* abc */ && wrong === 'never' /* second comment */) {
```

Reply:
Ah, yes, that will fail because there are multiple nodes for the same line. How embarassing. 
I guess I will change that to a set of lines, similar to #32.

Reply:
You only have to memorize the last line. If the start of the next one is the same as the end of the last, substract one.

Reply:
Mh, that could also work, yes. 
However, I am unsure if I can rely on that the results/matches of the query are ordered regarding the lines of code.

Reply:
I am confident that is the case, you can also verify that with a test and use the faster implementation as long as the test is green.

Reply:
Seems to work with the less computational-intensive solution suggested by you, so I have changed that accordingly